experimental/FTheoryTools/docs/src/literature.md

```diff
@@ -13,7 +13,7 @@ su5_tate_model_over_arbitrary_3d_base()
 ```
 More generally, we support literature constructions.
 ```@docs
-literature_model(; doi::String="", arxiv_id::String="", version::String="", equation::String="", model_parameters::Dict{String,<:Any} = Dict{String,Any}(), base_space::FTheorySpace = affine_space(NormalToricVariety, 0), model_sections::Dict{String, <:Any} = Dict{String,Any}(), completeness_check::Bool = true)
+literature_model(; doi::String="", arxiv_id::String="", version::String="", equation::String="", model_parameters::Dict{String,<:Any} = Dict{String,Any}(), base_space::FTheorySpace = affine_space(NormalToricVariety, 0), model_sections::Dict{String, <:Any} = Dict{String,Any}(), defining_classes::Dict{String, <:Any} = Dict{String,Any}(), completeness_check::Bool = true)
```

Comment:
I know we've already discussed this somewhat, but I want to further talk about the `model_sections` argument here. This has to be kept to agree with the book, but it now means something different than the "model_sections" appearing in the user-facing function `classes_of_model_sections`, correct? The `model_sections` argument here, I believe, corresponds to what is now called "defining classes" (without Kbar). Maybe this should be a named argument we keep for backwards compatibility but that goes undocumented?